sections/video.liquid

```diff
@@ -64,21 +64,23 @@
             class="video-section__poster media media--transparent{% if poster == null %} video-section__placeholder{% endif %}"
           >
             {%- if poster != null -%}
-              {{ poster
+              {{
+                poster
                 | image_url: width: 3840
-                | image_tag: loading: 'lazy', sizes: sizes, widths: '375, 750, 1100, 1500, 1780, 2000, 3000, 3840', alt: alt
+                | image_tag:
+                  loading: 'lazy',
+                  sizes: sizes,
+                  widths: '375, 750, 1100, 1500, 1780, 2000, 3000, 3840',
+                  alt: alt
               }}
             {%- else -%}
               {{ 'collection-2' | placeholder_svg_tag: 'placeholder-svg placeholder' }}
             {%- endif -%}
           </a>
         {%- else -%}
-          {{ section.settings.video
-            | video_tag:
-              image_size: "1100x",
-              loop: section.settings.enable_video_looping,
-              controls: true,
-              muted: false
+          {{
+            section.settings.video
+            | video_tag: image_size: '1100x', loop: section.settings.enable_video_looping, controls: true, muted: false
```

Comment:
Non blocking: I guess these are just prettier formatting? I suppose we may see a few of these now that formatOnSave is enabled by default.

Reply:
Exactly 👍